install_config/cluster_metrics.adoc

```diff
@@ -753,3 +753,291 @@ by performing the following steps:
 $ ansible-playbook <OPENSHIFT_ANSIBLE_DIR>/byo/openshift-cluster/openshift-metrics.yml \
    -e openshift_metrics_install_metrics=False
 ----
+
+[[openshift-prometheus]]
+== Prometheus on OpenShift
+
+Prometheus is a stand-alone, open source systems monitoring and alerting
+toolkit. You can use Prometheus to visualize metrics and alerts for OpenShift
+system resources.
+
+[IMPORTANT]
+====
+The Prometheus is a Technology Preview feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+[openshift-prometheus-roles]
+=== Setting Prometheus Role Variables
+
+The Prometheus role creates:
+
+* OpenShift-Prometheus namespace.
+* Prometheus clusterrolebinding and service account.
+* PVs for Prometheus, alertmanager and alertbuffer for internal NFS.
+* Prometheus pod with Prometheus behind oauth-proxy, alertmanager and alert-buffer behind oauth-proxy.
+* Prometheus and alertmanager configmaps.
+* Prometheus and alerts services and direct routes.
+* Prometheus, alertmanager and alert-buffer PVCs.
+
+Depends on `openshift_hosted_prometheus_deploy` flag.
+
+Set the following role variables to install and configure Prometheus.
+
+.Prometheus Variables
+[options="header"]
+|===
+
+|Variable |Description
+
+|`openshift_prometheus_state`
+|Set variable to install and update. Remove variable to uninstall.
+
+|`openshift_prometheus_namespace`
+|Project namespace where the components will be deployed. Default set to
+`openshift-prometheus`.
+
+|`openshift_prometheus_replicas`
+|The number of replicas for prometheus deployment. The default is one. Multiple
+replicas are not supported.
+
+|`openshift_prometheus_node_selector`
+|Selector for the nodes on which prometheus will be deployed.
+
```

Comment:
design is changed, we don't have the following parameters now
openshift_prometheus_image_proxy
openshift_prometheus_image_prometheus
openshift_prometheus_image_alertmanager
openshift_prometheus_image_alertbuffer

you can contact the developers to get the details

Reply:
I commented these parameters out of the document. Do they also need to be removed in the install examples?

Reply:
yes, to be removed, since  they are no longer in the final implementation. #